src/chrome-launcher.ts

```diff
@@ -167,6 +167,10 @@ class Launcher {
     return DEFAULT_FLAGS.slice();
   }
 
+  static getInstallations() {
+    return chromeFinder[getPlatform() as SupportedPlatforms]();
```

Comment:
note: `await` was previously used here; however, afaict, `chromeFinder[]()` methods intentionally use blocking I/O and synchronous interfaces --I opted to implement `getInstallations()` as _non_-async to avoid any ambiguity to the caller as to whether or not the method was blocking.

happy to change this if y'all feel differently 😄